src/ansys/fluent/core/session_base_meshing.py

```diff
@@ -146,10 +147,20 @@ def workflow(self):
         return self._old_workflow
 
     @property
-    def new_workflow(self):
+    def new_wt_workflow(self):
```

Comment:
Since we are always returning a stored instance, new_ is not appropriate. We can drop that part and expand the contraction: `watertight_workflow`.